src/main/fc/cli.c

```diff
@@ -683,7 +687,8 @@ static void cliAux(char *cmdline)
             if (ptr) {
                 val = fastA2I(ptr);
                 if (val >= 0 && val < CHECKBOX_ITEM_COUNT) {
```

Comment:
This will fail for permanent ID > 41 (they go up tp 50). Please find another way of testing validity.

Reply:
e.g. by removing the CHECKBOX_ITEM_COUNT and instead checking for a NULL return from findBox...